CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ All notable changes to [bpmn-auto-layout](https://github.com/bpmn-io/bpmn-auto-l
 
 _**Note:** Yet to be released changes appear here._
 
+## 1.0.0
```

Comment:
Any reason you bump this up to 1.0? Do you deem it mature enough now? For semver 0.5.0 is an OK the next major version.

Reply:
Updated to v0.5.0 in the PR.

Reply:
> Any reason you bump this up to 1.0? Do you deem it mature enough now? 

I suppose that migration necessity from the old node.js version to the new one is a breaking change point in the most cases. Because the latest packages versions (from dependencies list) can't work correctly with the unsupported node version.

NB. As example, I found [the `webpack-merge` package changelog](https://github.com/survivejs/webpack-merge/blob/develop/CHANGELOG.md#600--xxxx-xx-xx). And I think, `bpmn-auto-layout` has a similar case

Reply:
@nikku, what do you think about it?

Reply:
Thanks for the explanation